processing/src/test/java/org/apache/druid/math/expr/FunctionTest.java

```diff
@@ -952,6 +952,15 @@ public void testComplexDecode()
         ),
         expected
     );
+    // test with alias
```

Comment:
Nit. Might be good to add a test with 2 aliases, just to drive the point home that we can support multiple aliases for the same

Reply:
added test to verify multiple aliases